portal-impl/src/com/liferay/portal/service/impl/RoleLocalServiceImpl.java

```diff
@@ -1874,6 +1860,31 @@ public void unsetUserRoles(long userId, long[] roleIds)
 		reindex(userId);
 	}
 
+	@Override
+	public Role updateExternalReferenceCode(
+			long roleId, String externalReferenceCode)
```

Comment:
I'm not sure ERC should be the second parameter

Reply:
it's very "random"
```
public OAuth2Application updateExternalReferenceCode(
			long oAuth2ApplicationId, String externalReferenceCode)
```
but also
```
public CommercePriceEntry updateExternalReferenceCode(
			String externalReferenceCode, CommercePriceEntry commercePriceEntry)
```
but also
```
public CommerceTierPriceEntry updateExternalReferenceCode(
			CommerceTierPriceEntry commerceTierPriceEntry,
			String externalReferenceCode)
```